server/internal/store/user_org.go

```diff
@@ -0,0 +1,11 @@
+package store
+
+import "gorm.io/gorm"
+
+// UserOrganization is a model for user_organization
+type UserOrganization struct {
+	gorm.Model
+
+	User         uint `gorm:"uniqueIndex:user_org"`
```

Comment:
Just wanted to make sure. The value here is the database ID?

Reply:
Yes, it's the database ID, not the value used in RPC calls.